pages/index.js

```diff
@@ -3,7 +3,7 @@ import styles from '../styles/Home.module.css';
 import React, { useState } from 'react';
 import Image from 'next/image';
 
-import teduardoImage from 'C:\\!PROJEKTY_PRYWATA\\ted-and-agata\\pages\\images\\Ted_As_Coach_Bert.png';
+import teduardoImage from 'C:\!PROJEKTY_PRYWATA\ted-and-agata\public\Ted_As_Coach_Bert.png';
```

Comment:
You can't reference a file from your own hard drive. In NextJS, any files in the public folder are available by referencing the *root* application like this: `/Ted_As_Coach_Bert.png`

```suggestion
import teduardoImage from '/Ted_As_Coach_Bert.png';
```